### DISCUSSION TITLE
Getting an error when trying to run the start_web.bat file, Failed to import optimum.onnxruntime.modeling_seq2seq error

### DISCUSSION THREAD

**Discussion by Schorakbi** [OleehyO/TexTeller#77] [Q&A] [answered]
Hi, i installed today TexTeller and installed all dependencies and everything the command `python inference.py -img "C:\Users\schor\Downloads\ilovepdf_pages-to-jpg\math_pages-to-jpg-0001.jpg" -mix ` runs well and gets ouput but whenever i want to run the web app i get this error : `RuntimeError: Failed to import optimum.onnxruntime.modeling_seq2seq because of the following error (look up to see its traceback): Failed to import optimum.onnx.graph_transformations because of the following error (look up to see its traceback): DLL load failed while importing onnx_cpp2py_export: A dynamic link library (DLL) initialization routine failed.`
![image](https://github.com/user-attachments/assets/071c01ca-d1c8-4216-a298-24b68f1f04dd)

**Comment by OleehyO** [comment, marked as answer]
It seems like an environment configuration issue (unable to use ONNX inference properly). I have modified the source code in [TexTeller.py](https://github.com/OleehyO/TexTeller/blob/main/src/models/ocr_model/model/TexTeller.py) to better handle the dependencies. Please update your local repository and check if the issue persists.

**Reply by Schorakbi** [reply]
Just it launched i Can run it but i can't run it with oxxnruntime enabled
